sktime/dists_kernels/compose_tab_to_panel.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+"""
+Composers that create panel pairwise transformers from table pairwise transformers
+"""
+
+__author__ = ["fkiraly"]
+
+import numpy as np
+
+from sktime.dists_kernels._base import BaseTrafoPwPanel
+
+
+class AggrDist(BaseTrafoPwPanel):
+    """
+    pairwise distance based on aggregation function applied to sample-wise distance
+        example: AggrDist(ScipyDist()) is mean Euclidean distance between series
+
+    Components
+    ----------
+    transformer: pairwise transformer of BaseTrafoPw scitype (tabular pairwise)
```

Comment:
the fact you included '(tabular pairwise)' is indication that having 'tabular' as part of the baseclass name would be helpful

Reply:
I don't know, I would find it confusing - better have a base concept, and then `Panel` is appended for modifying the meaning.

Reply:
> > Does that mean you don't like BasePairwise?
> 
> Pairwise what? I find this not descriptive enough and slightly misleading

I agree that Pairwise byitself leaves something missing, but:
* Pairwise is used in sklearn (does not mean it is a good name, but it *is* an advantage of Pairwise)
* I think PairwiseTransformer is more misleading than just Pairwise, because it makes it sound like it is a similar kind of thing as the other Transformers in sktime, which I dont think it is.

Reply:
further discussed in comment below